tests/disjoint_bcast_debug/main.cpp

```diff
@@ -0,0 +1,83 @@
+#include "enrico/comm.h"
```

Comment:
Did you want to keep this test around or as it just for debugging purposes?

Reply:
Let's discard this test driver.  Like you noted, it doesn't compile now.  I think it was intended to test an earlier version of the code.  Furthermore, the info reported by this test driver (the communicator layouts) is now reported by ENRICO itself during normal runs.

Reply:
Deleted this driver.